.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Create GitHub release
```

Comment:
I'd like this workflow to be steps in the `deploy` workflow, or else called specifically by the `deploy` workflow (rather than `on.push` etc.). 

Otherwise, we get the GitHub release published well before the code is actually deployed, which feels a little out of order.

Reply:
Thanks! I agree, the `release` job should run after the `deploy` job is finished, so I'll add some steps in `deploy.yml` to achieve this. Oh and `release` should only run on an actual release, not a release candidate so I'll be careful about keeping this behavior unchanged.